docs/conf.py

```diff
@@ -98,9 +98,47 @@
 source_suffix = {'.rst': 'restructuredtext','.md': 'markdown'}
 
 # Substitution variables
+def module_version(module, version):
+    if module == 'github.com/intel/goresctrl':
+        version = '.'.join(version.split('.')[0:2]) + '.0'
+    return version
+
+def gomod_versions(goModPath, modules):
+    versions = {}
+    in_requires = False
+    with open(goModPath) as f:
+        for line in f:
+            line = line.strip()
+            if line == "" or line.startswith("//"):
+                continue
+            if not in_requires:
+                if line.startswith('module '):
+                    continue
+                elif line.startswith('require ('):
+                    in_requires = True
+                    continue
+                if line.startswith('go '):
+                    versions['golang'] = line[3:]
+                    continue
+                if line.startswith('replace ('):
+                    break
+            else:
+                if line.startswith(')'):
+                    in_requires = False
+                    continue
+                name, version = line.split(' ', 1)[0:2]
+                path = name.split('/')
+                if path[-1].startswith('v') and path[-1][1:].isdigit():
+                    name = '/'.join(path[0:-1])
+                if name in modules:
+                    version = version.split('-', 1)[0].split('//', 1)[0].strip()
+                    versions[name] = module_version(name, version)
+    return versions
```

Comment:
Woah, looks long and error prone 😎 Could we utilize `go list` here, e.g. something like
```
go list -m -f {{.Version}} <module-name>
```
should give you the version

Reply:
Yeah, I'll update to use that and `go list -m -f '{{.GoVersion}}'` for golang version.

Reply:
Okay, I rewrote it to use `subprocess.run(['go', 'list', '-m', '-f', ...])`.